src/main/java/mk/learning/fileshare/Services/ReadExcel.java

```diff
@@ -0,0 +1,47 @@
+package mk.learning.fileshare.Services;
+
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.util.HashMap;
+import java.util.Map;
+
+import org.apache.poi.ss.usermodel.Cell;
+import org.apache.poi.ss.usermodel.DataFormatter;
+import org.apache.poi.ss.usermodel.Row;
+import org.apache.poi.ss.usermodel.Sheet;
+import org.apache.poi.ss.usermodel.Workbook;
+import org.apache.poi.xssf.usermodel.XSSFWorkbook;
+import org.springframework.stereotype.Component;
+
+@Component
+public class ReadExcel {
+
+	public static Map<String, Map<String, String>> setMapData(String Filename) throws IOException {
+		DataFormatter formatter = new DataFormatter();
+		String path = Filename;// "E:\testshare.xlsx";
+		FileInputStream inputStream = new FileInputStream(path);
+		Workbook workbook = new XSSFWorkbook(inputStream);
+		Sheet sheet = workbook.getSheetAt(0);
+		int LastRow = sheet.getLastRowNum();
+		Map<String, Map<String, String>> ExcelMapData = new HashMap<String, Map<String, String>>();
```

Comment:
This Should be:
if(Constants.mapOfMaps == null){
Constants.mapOfMaps = new HashMap<String,HashMap<String,String>>();
}